source/fundamentals/connection.txt

```diff
@@ -7,7 +7,7 @@ Connection Guide
 .. default-domain:: mongodb
 
 .. facet::
-  :type: genre 
+  :name: genre 
```

Comment:
should these be indented another space? same for other file too.

Reply:
Yep!  I'll adjust.